api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/calculateCustomSummary.md

```diff
@@ -217,6 +217,8 @@ A summary value calculation is conducted in three stages: *start*, in which the
 
 ---
 
+[note]If you need to access all properties of each cell value, do not specify the [dataField](/Documentation/ApiReference/Data_Layer/PivotGridDataSource/Configuration/fields/#dataField) option for the field that contains this function.
```

Comment:
I think the `options.value` description is a more suitable place for this information.